dexter/articles.py

```diff
@@ -164,10 +198,33 @@ def edit_article(id):
 
 @app.route('/articles/<id>/analysis', methods=['GET', 'POST'])
```

Comment:
likewise, I'd move this to `/articles/<id>/analysis/fdi` so that you're not having to interleave the form parsing code.